supabase/sql/advisor_drop_duplicate_indexes.sql

```
@@ -0,0 +1,19 @@
+-- ITX-52: drop redundant duplicate indexes (advisor 0009_duplicate_index).
+-- The `create index` statements for the first three were removed from their
+-- source migrations (district_foundation.sql, performance_tuning.sql); this
+-- drops the copies already deployed. The kept index of each pair is a PK or
+-- UNIQUE index covering the same columns. Supabase-managed auth.*/storage.*
+-- duplicates are intentionally left alone.
+
+-- (slug) — duplicates unique districts_slug_key
+drop index if exists public.idx_districts_slug;
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)